### ISSUE TITLE
Silence "Speeddating plugin not installed"

### ISSUE THREAD

**Issue by rgrinberg** [jceb/vim-orgmode#203]
I'm not interested in this plugin or the integration vim-orgmode provides with it. Is it possible to disable this message somehow?

**Comment by jceb** [comment]
No, currently it's not possible to disable the message.  Try to define the commands `:SpeedDatingFormat` and `:1SpeedDatingFormat` and let them accept arbitrary arguments.  This should silence the warning.

**Comment by andschwa** [comment]
I would also be interested in this :)

But I am very glad that this plugin exists. Org mode is what moved me from Vim to Emacs originally!

**Comment by chrahunt** [comment]
Example which can be put in vimrc:

```
command -nargs=* -range SpeedDatingFormat
```
